### PR TITLE
style(storefront-twbs): bs5 styles backport

### PR DESCRIPTION
**Pull request by sameoldcarlos**
**Summary**
Fit bootstrap elements styles to new standards of version 5

**Screenshots**
image
image
image

**Checklist**

- [x] I've read and understood [contributing guidelines](https://github.com/ecomplus/storefront/blob/master/CONTRIBUTING.md);
- [x] If changing UI, I've tested on most common viewports, desktop and mobile devices;

**A picture of a cute animal**
image

### CODE REVIEW

**Review by leomp12 — APPROVED** [review]
valeu @sameoldcarlos 💜 🚀

File: @ecomplus/storefront-template/template/scss/components/_close.scss [hunk 1]
```diff
@@ -0,0 +1,48 @@
+// Transparent background and border properties included for button version.
+// iOS requires the button element instead of an anchor tag.
+// If you want the anchor version, it requires `href="#"`.
+// See https://developer.mozilla.org/en-US/docs/Web/Events/click#Safari_Mobile
+
+.close {
+  float: right;
+  font-size: 1.5rem;
+  line-height: 1;
+}
```
Comment (leomp12) [inline @ecomplus/storefront-template/template/scss/components/_close.scss:10]: Tem só um ponto aqui que eu vi agora, na verdade acho que esse arquivo deveria estar no `storefront-twbs` mesmo, e importado no `styles.scss` dele lá...

Comment (leomp12) [reply @ecomplus/storefront-template/template/scss/components/_close.scss:10]: 🙃

Comment (sameoldcarlos) [reply @ecomplus/storefront-template/template/scss/components/_close.scss]: Ah sim, verdade
Movi lá agora

**Review by leomp12 — APPROVED** [review]
valeu @sameoldcarlos 👍🏾 💜

**Pull request merged**